massgov/mayflower
# DP-10131 Scrollbar on contact list in IE

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Remove scroll bars displayed on contact list in IE.
Add IE specific css to resolve the issue.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-10131)

## Steps to Test


TEST IN IE IN DRUPAL SINCE MAYFLOWER USES A DIFFERENT TEMPLATE FOR THE CONTACT LIST CONTAINER. 

1.  Go to below sample pages and check if scroll bars are displayed for the contact list in IE.  Also, check in other browsers for any odd displays.

**Main column:**
- /orgs/executive-office-of-energy-and-environmental-affairs
- /orgs/department-of-industrial-accidents
- /orgs/division-of-fisheries-and-wildlife

**Side column:**
- /service-details/procedures-for-conducting-prospective-concurrent-and-retrospective-utilization
- /service-details/learn-about-black-bears
- /service-details/check-eligibility-for-unemployment-benefits

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
